packages/grafana-ui/src/components/Table/FilterList.tsx

```diff
@@ -2,33 +2,131 @@ import { css, cx } from '@emotion/css';
 import React, { useCallback, useMemo, useState } from 'react';
 import { FixedSizeList as List } from 'react-window';
 
-import { GrafanaTheme2, SelectableValue } from '@grafana/data';
+import { GrafanaTheme2, formattedValueToString, getValueFormat, SelectableValue } from '@grafana/data';
 
-import { Checkbox, FilterInput, Label, VerticalGroup } from '..';
+import { ButtonSelect, Checkbox, FilterInput, HorizontalGroup, Label, VerticalGroup } from '..';
 import { useStyles2, useTheme2 } from '../../themes';
 
 interface Props {
   values: SelectableValue[];
   options: SelectableValue[];
   onChange: (options: SelectableValue[]) => void;
   caseSensitive?: boolean;
+  showOperators?: boolean;
 }
 
 const ITEM_HEIGHT = 28;
 const MIN_HEIGHT = ITEM_HEIGHT * 5;
 
-export const FilterList = ({ options, values, caseSensitive, onChange }: Props) => {
+const operatorSelectableValue = (op: string): SelectableValue<string> => {
+  const result: SelectableValue<string> = { label: op, value: op };
+  switch (op) {
+    case 'Contains':
+      result.description = 'Contains';
+      break;
+    case '=':
+      result.description = 'Equals';
+      break;
+    case '!=':
+      result.description = 'Not equals';
+      break;
+    case '>':
+      result.description = 'Greater';
+      break;
+    case '>=':
+      result.description = 'Greater or Equal';
+      break;
+    case '<':
+      result.description = 'Less';
+      break;
+    case '<=':
+      result.description = 'Less or Equal';
+      break;
+    case 'Expression':
+      result.description =
+        'Bool Expression (Char v represents the column value in the expression, e.g. "v >= 10 && v <= 12")';
```

Comment:
Variables in Grafana are typically prepended with a `$` and I think we should use that here as well

Reply:
Done, thanks @codeincarnate !!!!